### ISSUE TITLE
alernativa to 0.2.9 parse()?

### ISSUE THREAD

**Issue by boxxxie** [swannodette/mori#136]
i see that mori's EDN parser is gone now, is there a seperate npm module i can rely on for that functionality?

**Comment by boxxxie** [comment]
actually, i'm using mori on nodejs, and i really don't care about dead code removal... it would be nice to have a web build and a nodejs build, or even a new repo/fork.

**Comment by swannodette** [comment]
@boxxxie we'll get there eventually. Going to have to deal with a little bit of churn in the meantime until we can get enough support from ClojureScript to proper builds optimized for multiple targets. In the meantime I can strongly recommend using transit-js for this, it's way faster than parse, 20-30X.

**Comment by orlin** [comment]
It may be a silly question, yet I didn't get it from web-search or anything else -- whether I can parse an edn string (into JSON) through transit...  Using JavaScript to call the Datomic REST server, for example.  It had been recommended to use jsedn.  Still the way?

**Comment by swannodette** [comment]
`parse` is coming right back once the modular build stuff lands which underway now. We just need adequate support from ClojureScript itself, see http://dev.clojure.org/jira/browse/CLJS-1188
